src/components/Workspace.jsx

```diff
@@ -268,7 +188,7 @@ class Workspace extends React.Component {
             ref={this._storeDividerRef}
           />
         </DraggableCore>
-        {this._renderOutput()}
+        <Output />
```

Comment:
`<Output onRef={this._handleOutputRef} />`

`onRef` is an “own prop” that is passed through to the `Output` presentational component.